build/zig/build.sh

```diff
@@ -27,7 +27,7 @@ set_arch 64
 # We want to populate the clang-related environment variables
 # and set PATH to point to the correct llvm/clang version
 # but we want to build with gcc for releases before r151041
-set_clangver 13 # zig 0.9.x requires LLVM 13
+set_clangver 16 # zig 0.11.x requires LLVM 16
```

Comment:
LLVM 16 is only available in r151045 and higher, so can you add a `min_rel` call so that it just gets skipped on earlier releases?

That would be just `minrel 151045`, before `set_arch` for consistency with other scripts.

@hadfl do you think we should preserve the build script for 0.9.1 for those older releases?

Reply:
Thanks, I also removed the `test_relvar` logic around 151041 since it doesn't seem to apply anymore.

I'm happy to add 0.9.1 back in. Though keep in mind it's an outdated version of the language and stdlib -- Zig has changed a lot in those two point releases.